apps/explorer/src/components/OwnedObjects/index.tsx

```diff
@@ -34,14 +33,9 @@ export function OwnedObjects({ id }: { id: string }) {
 
 	return (
 		<div className="flex flex-col gap-4 pt-5">
-			<RadioGroup
-				className="flex"
-				ariaLabel="transaction filter"
-				value={filter}
-				onChange={setFilter}
-			>
+			<RadioGroup aria-label="transaction filter" value={filter} onValueChange={setFilter}>
```

Comment:
I know you didn't write this label and don't blame you if you don't want to change it in this PR, but a more descriptive label would be nice haha. Maybe `View transactions by a specific filter` or something along those lines